webapp/components/admin_console/elasticsearch_status/status.jsx

```diff
@@ -0,0 +1,359 @@
+                status = (
+                    <FormattedMessage
+                        id='admin.elasticsearchStatus.statusInProgress'
+                        defaultMessage='Job in progress. {percent}% complete.'
+                        values={{
+                            percent: chosenJob.progress
+                        }}
+                    />
+                );
+                statusHelp = (
+                    <FormattedMessage
+                        id='admin.elasticsearchStatus.statusInProgress.help'
+                        defaultMessage='Indexing is in progress on the job server. If Elasticsearch is enabled, search results may be incomplete until the job is finished.'
+                    />
+                );
+                statusColor = '#eac262';
+            } else if (chosenJob.status === JobStatuses.SUCCESS) {
+                status = (
+                    <FormattedMessage
+                        id='admin.elasticsearchStatus.statusSuccess'
+                        defaultMessage='Indexing complete.'
+                    />
+                );
+                statusHelp = (
+                    <FormattedMessage
+                        id='admin.elasticsearchStatus.statusSuccess.help'
+                        defaultMessage='Indexing is complete and new posts are being automatically indexed.'
+                    />
+                );
+                statusColor = '#69c169';
+            } else if (chosenJob.status === JobStatuses.ERROR) {
+                status = (
+                    <FormattedMessage
+                        id='admin.elasticsearchStatus.statusError'
+                        defaultMessage='Indexing error.'
+                    />
+                );
+                statusHelp = (
+                    <FormattedMessage
+                        id='admin.elasticsearchStatus.statusError.help'
+                        defaultMessage='Mattermost encountered an error building the Elasticsearch index: {error}'
+                        values={{
+                            error: chosenJob.data ? (chosenJob.data.error || '') : ''
+                        }}
+                    />
+                );
+                statusColor = '##ea6262';
+            } else if (chosenJob.status === JobStatuses.CANCEL_REQUESTED) {
+                status = (
+                    <FormattedMessage
+                        id='admin.elasticsearchStatus.statusRequestCancel'
+                        defaultMessage='Canceling Job...'
+                    />
+                );
+                statusColor = '#eac262';
+            } else if (chosenJob.status === JobStatuses.CANCELED) {
+                status = (
+                    <FormattedMessage
+                        id='admin.elasticsearchStatus.statusCancelled'
+                        defaultMessage='Indexing job cancelled.'
+                    />
+                );
+                statusColor = '##ea6262';
```

Comment:
Typo error here: `##ea6262`

Would it be possible to move this hardcoded `statusColor` somewhere to *.scss? e.g.: 
```
.job-cancel {
    color: #ea6262;
} 
```